src/TestAppCore/MemoryStreamExtensions.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Licensed under the BSD license
+// See the LICENSE file in the project root for more information
+
+using System;
+using System.IO;
+
+namespace TestApp
+{
+    internal static class MemoryStreamExtensions
+    {
+        public static byte[] GetBuffer(this MemoryStream stream)
+        {
```

Comment:
When I made this comment I did not notice that source files using GetBuffer were shared by the two test apps